doc/api/crypto.md

```diff
@@ -1961,6 +1961,25 @@ is currently in use. Setting to true requires a FIPS build of Node.js.
 This property is deprecated. Please use `crypto.setFips()` and
 `crypto.getFips()` instead.
 
+### `crypto.checkPrime(candidate[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `candidate` {ArrayBuffer|SharedArrayBuffer|TypedArray|Buffer|DataView} A
+  possible prime encoded as a sequence of big endian octets of arbitrary
+  length.
+* `options` {Object}
+  * `checks` {number} The number of primality checks to perform.
+    **Defaults**: `1`
+  * `fast` {boolean} `true` to use the fast check algorithm.
```

Comment:
I think the documentation should say what is the trade-off of using the fast algorithm

Reply:
I've actually removed the fast option. It just adds complexity that's difficult to explain around.